Frontend/Pages/AccessibilityStatement.cshtml

```diff
@@ -7,96 +7,112 @@
-            <li>the web address (URL) of the content</li>
-            <li>your name and email address</li>
-            <li>the format you need, for example, audio CD, braille, BSL or large print, accessible PDF</li>
-        </ul>
-        <h3 class="govuk-heading-m">Enforcement procedure</h3>
-        <p class="govuk-body govuk-!-margin-bottom-7">
-            The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies
-            (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).
-            If you’re not happy with how we respond to your complaint,
-            <a href="https://www.equalityadvisoryservice.com" target="_blank" class="govuk-link">
-                contact the Equality Advisory and Support Service (EASS) (opens in new tab).
-            </a>
-        </p>
-        <h2 class="govuk-heading-l">Contacting us by phone</h2>
-        <p class="govuk-body govuk-!-margin-bottom-7">
-            If you are having problems using this service and you need help,
-            call the Department for Education on 0370 000 2288.
-        </p>
-        <h2 class="govuk-heading-l">Technical information about this website's accessibility</h2>
-        <p class="govuk-body govuk-!-margin-bottom-7">
-            The Department of Education (DfE) is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
-        </p>
-        <h3 class="govuk-heading-m">Compliance status</h3>
-        <p class="govuk-body govuk-!-margin-bottom-7">
-            This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed.
-        </p>
-        <h2 class="govuk-heading-l">Non-accessible content</h2>
-        <p class="govuk-body govuk-!-margin-bottom-7">The content listed is non-accessible for the following reasons.</p>
-        <h3 class="govuk-heading-m">Non-compliance with the accessibility regulations</h3>
-        <p class="govuk-body">
-            Main headings on some pages are contained within groups of elements and so may not be read out correctly by some screen reader software.
-            This fails WCAG 2.1 AA success criteria 2.4.6 (Headings and Labels).
-        </p>
-        <p class="govuk-body">
-            Some labelling on primary action buttons are not consistent. The labels were selected to give users a better description of the purpose of the action based on user research.
-            This fails WCAG 2.1 AA success criteria 3.2.4 (Consistent Identification).
-        </p>
-        <p class="govuk-body govuk-!-margin-bottom-7">
-            We are looking into the two issues mentioned previously and aim to resolve these in a future iteration of the service.
-        </p>
+<h1 class="govuk-heading-xl">
```

Comment:
The indentation has gone awry here.

Reply:
Sorry it's careless from me :) Just updated.